internal/controllers/machine_controller.go

```diff
@@ -244,6 +252,9 @@ func (r *MachineReconciler) startCheckAndEnqueueVolumeResize(ctx context.Context
 				}
 
 				if lastVolumeSize := getLastVolumeSize(machine, GetUniqueVolumeName(plugin.Name(), volumeID)); volumeSize != ptr.Deref(lastVolumeSize, 0) {
+					if eventErr := r.eventStore.AddEvent(machine.Metadata, corev1.EventTypeNormal, "SizeChangedVolume", fmt.Sprintf("Volume size changed %s. lastVolumeSize: %d. volumeSize: %d", volume.Name, *lastVolumeSize, volumeSize)); eventErr != nil {
```

Comment:
could you pls maybe add here unit.

Reply:
and maybe it will be better replace . with comma because it looks as list of attributes

Reply:
done